pages/task/list/views/tasklist.jsx

```diff
@@ -68,9 +70,24 @@ const Tasklist = ({
     );
   }
 
+  progress = progress || tabs[0];
+  const selected = tabs.indexOf(progress);
+
   return (
     <Fragment>
-      <StatusFilters tabs={tabs} progress={progress} />
+      {
+        displayTabs
+          ? (
+            <Fragment>
+              {
+                !!tabs.length && <Tabs active={selected}>
```

Comment:
It might be good to add this condition to the component so we can get rid of the extra cruft to handle the (probably impossible) zero tab case..